tests/test_storageclass_deletion.py

```diff
@@ -0,0 +1,151 @@
+"""
+Automates te following test:-
+https://polarion.engineering.redhat.com/polarion/#/project/OpenShiftContainerStorage/workitem?id=OCS-297
```

Comment:
@RazTamir is it ok, to expose internal link in upstream project?

Reply:
@dahorak yes, as no one has credentials it is ok for now.
But your concern is valid - as soon as we will split our tests to tier1 (upstream for gate testing) and tier2+ (downstream for regression cycle testing) we will need to remove those internal link for tier1 tests

Reply:
+1 for the suggestion. Hence, removed